src/slides/slides.css

```diff
@@ -523,8 +529,14 @@ body:lang(en) {
 .reveal .slides footer {
   position: absolute;
   bottom: var(--footer-vertical-margin);
-  font-size: 12px;
-  opacity: 0.5;
+  /* unset styles from Reveal */
+  background-color: unset;
+  color: unset;
+  font-family: unset;
+  font-size: unset;
```

Comment:
Useless with the `font-size` 3 lines after.

Reply:
Done